<?xml version="1.0"?>
<launch>
    <!-- Parameters -->
    <arg name="tf_prefix"             default="oak"                                 />
    <arg name="camera_param_uri"      default="package://tht_depthai/params/camera" />    
    <arg name="manual_focus"          default="120"                                 />
    
    <!-- RGB Publisher Node -->
    <node name="tht_rgb_node" pkg="tht_depthai" type="tht_rgb_node" output="screen" required="true">
        <param name="tf_prefix"        value="$(arg tf_prefix)"/>
        <param name="camera_param_uri" value="$(arg camera_param_uri)"/>
        <param name="manual_focus" value="$(arg manual_focus)"/>
    </node>
</launch>
